### PR TITLE
Added the missing settings' descriptions

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: docs/modules/ROOT/pages/operations.adoc [hunk 1]
```diff
@@ -339,12 +340,12 @@ The Bolt+routing feature on Neo4j drivers discovers a cluster member (drivers ha
 
 | `editorLint`
 | `false`
-| TODO
+| Enable/disable squiggly lines under text.
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/operations.adoc:343]: This is what the setting is supposed to do, but it doesn't work

File: docs/modules/ROOT/pages/operations.adoc [hunk 2]
```diff
@@ -292,22 +292,23 @@ Possible values: `auto`, `normal`, `outline`, and `dark`.
...
 | `500`
-| The maximum number of items per field.
+| Limit the length of the returned lists.
+For example, if a node has 1000 labels, the list will be cut off to the value set in `maxFieldItems`.
 | no
 
 | `shouldReportUdc`
 | `true`
-| TODO
+| Report less user metrics.
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/operations.adoc:311]: This is what the setting is supposed to do, but it doesn't work

**Review by OskarDamkjaer — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/operations.adoc (hunk 2 above)
Comment (renetapopova) [reply docs/modules/ROOT/pages/operations.adoc:311]: yeah, but we can't say in the docs - sorry, it doesn't work.

File: docs/modules/ROOT/pages/operations.adoc (hunk 1 above)
Comment (renetapopova) [reply docs/modules/ROOT/pages/operations.adoc:343]: same as below

File: docs/modules/ROOT/pages/operations.adoc (hunk 2 above)
Comment (renetapopova) [reply docs/modules/ROOT/pages/operations.adoc:311]: In other words, these are descriptions of the settings. If they don't work, we should fix them, I guess 😄

Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/operations.adoc:311]: OK! Not sure if we're going to fix or remove them though

File: docs/modules/ROOT/pages/operations.adoc [hunk 3]
```diff
@@ -272,7 +272,7 @@ Possible values: `auto`, `normal`, `outline`, and `dark`.
 
 | `playImplicitInitCommands`
 | `true`
-| TODO: Commands that are run when connected to the graph.
+| Allow you to run some commands automatically on startup, for example, `:play start` and `:server connect`.
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/operations.adoc:275]: Perhaps clearer as `If set to true browser will automatically run some commands on startup, for example :play start and :server connect`

Comment (renetapopova) [reply docs/modules/ROOT/pages/operations.adoc:275]: Does it run the command that is set in `initCmd`?

Comment (renetapopova) [reply docs/modules/ROOT/pages/operations.adoc:275]: If so, I can rephrase it to "If `true`, Browser automatically runs the command set in `initCmd` on startup.

File: docs/modules/ROOT/pages/operations.adoc (hunk 2 above)
Comment (martin-neotech) [reply docs/modules/ROOT/pages/operations.adoc:311]: What is `Udc` ?

File: docs/modules/ROOT/pages/operations.adoc (hunk 3 above)
Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/operations.adoc]: yes

File: docs/modules/ROOT/pages/operations.adoc [hunk 4]
```diff
@@ -339,12 +340,12 @@ The Bolt+routing feature on Neo4j drivers discovers a cluster member (drivers ha
...
 | `editorLint`
 | `false`
-| TODO
+| Enable/disable squiggly lines under text.
 | no
 
 | `useCypherThread`
 | `true`
-| TODO
+| Run cypher queries through a webworker to increase performance.
```
Comment (martin-neotech) [inline docs/modules/ROOT/pages/operations.adoc:348]: | Run Cypher queries through a web worker (background thread) to increase performance.

File: docs/modules/ROOT/pages/operations.adoc [hunk 5]
```diff
@@ -292,22 +292,23 @@ Possible values: `auto`, `normal`, `outline`, and `dark`.
 
 | `browserSyncDebugServer`
 | `null`
-| TODO
+| Internal setting for debugging browser sync.
```
Comment (martin-neotech) [inline docs/modules/ROOT/pages/operations.adoc:295]: | Internal setting, used for debugging Browser Sync.

File: docs/modules/ROOT/pages/operations.adoc [hunk 6]
```diff
@@ -359,7 +360,7 @@ The Bolt+routing feature on Neo4j drivers discovers a cluster member (drivers ha
 
 | `showPerformanceOverlay`
 | `false`
-| TODO
+| Internal setting to see FPS/performance.
```
Comment (martin-neotech) [inline docs/modules/ROOT/pages/operations.adoc:363]: | Internal setting, used to show a performance overlay (FPS and memory usage).

**Review by martin-neotech — COMMENTED** [review]
Thanks. some comments.

File: docs/modules/ROOT/pages/operations.adoc (hunk 2 above)
Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/operations.adoc:311]: user data collection, but the setting is like i said not currently wired to anything. so it might as well stand for anything

**Review by martin-neotech — APPROVED** [review]
Thanks.

**Review by OskarDamkjaer — APPROVED** [review]
